scripts/variantstore/wdl/GvsExtractCohortFromSampleNames.wdl

```diff
@@ -8,7 +8,8 @@ import "GvsExtractCallset.wdl" as GvsExtractCallset
 workflow GvsExtractCohortFromSampleNames {
 
   input {
-    File cohort_sample_names
+    File? cohort_sample_names
+    Array[String]? cohort_sample_names_array
```

Comment:
maybe a comment here that `cohort_sample_names` will be ignored if `cohort_sample_names_array` is defined?

Reply:
+1